packages/ocs/block-pool/body.tsx

```diff
@@ -79,6 +86,64 @@ export const BlockPoolBody = (props: BlockPoolBodyPros) => {
     []
   );
 
+  const [data, loaded, loadError] = useK8sList(CephBlockPoolModel);
+
+  const existingNames = React.useMemo(
+    () => (loaded && !loadError ? data?.map((data) => getName(data)) : []),
+    [data, loaded, loadError]
+  );
+
+  const fieldRequirements = [
+    t('No more than 253 characters'),
+    t('Starts and ends with a lowercase letter or number'),
+    t('Only lowercase letters, numbers, non-consecutive periods, or hyphens'),
+    t('Cannot be used before'),
```

Comment:
will not block the PR on this, but this is not very intuitive... we should confirm with UX once...
something like this (only suggestion):
```suggestion
    t('An unique name for this resource'),
```
```suggestion
    t('Should be an unique within same Namespace'),
```

Reply:
I cannot change the copy as per your suggestion as it came from the the UX writing team. 
https://docs.google.com/document/d/1yWm6NSHKjFslC09NYw82G7_hFvyTTjR9nEeT88QG2E8/

Reply:
Okay... not an important issue... we can fix later if any QE/customer points this out...

Reply:
somwhere we are using "Cannot be used before" and somewhere "A unique name for the resource within a project", does not make sense to me :/
 "Cannot be used before" >> should be used for Cluster wide resources
 "A unique name for the resource within a project" >> should be used for Namespace wide resources.

Reply:
I cannot change the copy as per your suggestion as it came from the the UX writing team.
https://docs.google.com/document/d/1yWm6NSHKjFslC09NYw82G7_hFvyTTjR9nEeT88QG2E8/

Reply:
coming from UX does not mean it is 100% correct, that is what we need to discuss with UX and get it fixed...
but not blocking the PR on this for now... but we will have to send a follow up...
this is common understanding that:
```
"Cannot be used before" >> should be used for Cluster wide resources
"A unique name for the resource within a project" >> should be used for Namespace wide resources.
```

Reply:
OBC/BucketClass/BlockPool/NamespaceStore/BackingStore are namespace wide resource... there names are unique within a particular namespace, but we can create a new CR with same name in different namespace.

Reply:
StorageClass is a cluster wide resource... in this case name should always be unique globally...

Reply:
I've never said that because it came from UX it is correct. I cannot/should not change this myself. UX team need to make a decision on this, then update the document https://docs.google.com/document/d/1yWm6NSHKjFslC09NYw82G7_hFvyTTjR9nEeT88QG2E8/ so QA can test this accordingly and then I can submit a change.

Reply:
I agree, yup... that is why I said "we need to discuss with UX and get it fixed... but not blocking the PR on this for now"